doc/examples/tiled/lib/main.dart

```diff
@@ -1,14 +1,43 @@
 import 'package:flame/components/tiled_component.dart';
 import 'package:flame/game.dart';
-import 'package:flutter/widgets.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/position.dart';
+import 'package:flame/animation.dart';
+import 'package:flutter/widgets.dart' hide Animation;
+import 'package:tiled/tiled.dart' show ObjectGroup, TmxObject;
 
 void main() {
+  Flame.images.load('coins.png');
   final TiledGame game = TiledGame();
   runApp(game.widget);
 }
 
 class TiledGame extends BaseGame {
+  TiledComponent tiledMap;
+  Animation a;
   TiledGame() {
-    add(TiledComponent('map.tmx'));
+    tiledMap = TiledComponent('map.tmx');
+    add(tiledMap);
+    a = Animation.sequenced('coins.png', 8, textureWidth: 20);
```

Comment:
To use the animations components I said, just put the following code here on the constructor:

```dart
    final ObjectGroup obj = tiledMap.getObjectLayerByName("AnimatedCoins");
    if (obj == null) {
      return;
    }
    for (TmxObject obj in obj.tmxObjects) {
      final comp = AnimationComponent(30.0, 30.0, a);
      comp.x = obj.x.toDouble();
      comp.y = obj.y.toDouble();
      add(comp);
}      
```

Then you will not need to override the render method.